sites/dev/src/_pages/blog/1_block-design.mdx

```diff
@@ -0,0 +1,275 @@
+
+1.  Blocks should be visually simple, minimal, and blend in with their surroundings
+2.  Blocks should follow tried-and-true interaction design principles
+
+The first principle helps blocks feel at home in the applications that use them. Each application will have its own visual style and set of design conventions they follow; a design system they've worked hard to establish and keep consistent across their product.
+
+We've designed the protocol with this in mind; we allow applications to pass styling variables into blocks such as `primaryColour` and `fontFamily` (more on this below) which helps them match the visual styling of their environments.
+
+There's more to design than visual appearance though. Blocks need to behave predictably and consistently for the people using them. This means being both internally consistent, and consistent with the way most other websites on the web work. This principle is known as [Jakob's Law](https://www.nngroup.com/videos/jakobs-law-internet-ux/) - users spend 99% of their time on apps and websites other than yours, and they expect them to all work the same way. There is no need to reinvent the wheel or the dropdown menu.
+
+There is a much higher risk of inconsistent behaviour when every block in an application is designed and built by someone different, without any shared conventions or rules. We've all felt the frustration of trying to use an app that has 99 button colours, icons that change meaning across screens, and labels that sporadically switch between “cancel,” “discard”, and “delete.” These small frustrations add up and will quickly deplete a user's patience.
+
+Thankfully, you can avoid most of these usability problems by following our visual styling principles and best practices when you're creating blocks.
+
+## Styling Block Protocol blocks
+
+We've setup our [block starter template](https://github.com/blockprotocol/blockprotocol/tree/main/packages/block-template) with a couple of helpful CSS files:
+
+- A CSS reset file
+- A base styles file with a set of root variables you can adjust
+
+Our base styles file is lightly opinionated and sets up a styling base that will help your block look and feel similar to others.
+
+It comes with a default palette of colours you can use, a typographic scale, a spacing scale, and other similar base styles. You can read more details about what the base stylesheet includes in our [documentation](#).
+
+### Accepting styles from embedding applications
+
+The blocks you build will be used inside other applications; we call these "embedding application."
+
+We designed the protocol to give embedding applications control over the way blocks appear in their product. We give embedding applications the final say over how a block looks, as they have more context about the environment it lives within and how people will use it.
+
+Every Block Protocol block you build must accept a `styleVariables` object that the embedding application can pass in.
```

Comment:
```suggestion
Every Block Protocol block you build should accept a `styleVariables` object that the embedding application can pass in.
```

I believe this is likely to be finalized as a _should_, and not a _must_ in the spec (https://blockprotocol.org/spec/appendix-a-styling). cc @CiaranMn